build-scripts/clang-tidy.sh

```diff
@@ -97,9 +97,16 @@ else
         includes
 
     tidyable_cpp_files="$( \
-        ( test -f ./files_changed && ( build-scripts/get_affected_files.py ./files_changed | grep -v third-party ) ) || \
+        ( test -f ./files_changed && ( build-scripts/get_affected_files.py ./files_changed ) ) || \
         echo unknown )"
 
+    tidyable_cpp_files="$(echo -n "$tidyable_cpp_files" | grep -v )"
```

Comment:
Wait, grep -v what? Should this still say `grep -v third-party`?

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/pull/78279

Reply:
Yes it defibiteoy should >_<